lms/static/js/instructor_dashboard/util.js

```diff
@@ -379,7 +379,7 @@
                 url: listEndpoint,
                 success: function(data) {
                     if (data.tasks.length) {
-                        ths.createTaskListTable(ths.$table_running_tasks, data.tasks);
+                        createTaskListTable(ths.$table_running_tasks, data.tasks);
```

Comment:
So what is [`this.createTaskListTable`](https://github.com/edx/edx-platform/blob/2ab3b9c852df3ca9f9762bc675e278012fb07929/lms/static/js/instructor_dashboard/util.js#L45) on line 45 for? With these changes, it's never called or exported.